build.sbt

```diff
@@ -215,16 +216,15 @@ lazy val packagingSettings = Seq(
   maintainer := "Mesosphere Package Builder <support@mesosphere.io>",
   serverLoading := None, // We override this to build for each supported system loader in the packageLinux alias
   debianPackageDependencies in Debian := Seq("java8-runtime-headless", "lsb-release", "unzip", s"mesos (>= ${Dependency.V.MesosDebian})"),
+  packageArchitecture in Debian := "amd64",
+  version in Debian := s"${version.value}-1.ubuntu1804",
```

Comment:
@timcharper, need a version for `debian8`, `debian9`, `ubuntu1404`, `ubuntu1604` and `ubuntu1804`. What is the best way to create these configurations?

Reply:
@jeschkies if these repos all include java8-runtime-headless, lsb-release, etc. then nothing is needed. We only create three deb packages, currently, one for each init system (systemv, SystemD, and Upstart). Then we publish the same package to each repo, according to their init system.

Reply:
Mappings for upload are here:  https://github.com/mesosphere/marathon/blob/v1.7.50/ci/releases.sc#L113